website/src/blog/2023-08-15-fusion/2023-08-15-fusion.md

```diff
@@ -0,0 +1,625 @@
+
+![Banana Cake Pop - Stages](bcp-3.png)
+
+But the core principle is that this is open and built into the GraphQL-Fusion spec. To provide tooling a single file containing all the information needed for gateway configuration and even space for gateway-specific features, we've adopted the [Open Packaging Convention](https://en.wikipedia.org/wiki/Open_Packaging_Conventions) as a container for the GraphQL-Fusion Configuration (.fgp).
+
+The [Open Packaging Convention](https://en.wikipedia.org/wiki/Open_Packaging_Conventions) is an open standard provided by [Microsoft](https://microsoft.com), used for everything from Word Documents (.docx) to VSCode extension packages (.visx). Simply put, think of it as a ZIP container with metadata and relations between its artifacts. The GraphQL-Fusion convention contains the mandatory Fusion Graph document. This document is all that's needed to run and configure a Gateway implementing the core specification.
+
+Additionally, it contains all subgraph schema documents, the publicly exposed Gateway schema, and composition settings the user has opted into. Having all these artifacts in one place gives us a single artifact that we can pass on from the schema composition in a CI/CD pipeline to the schema registry and from there to the actual gateway. We have customers already using this with their custom solutions for distributing the configuration from their deployment pipeline to their gateway or by using our Cloud Services ([Banana Cake Pop](https://eat.bananacakepop.com)). Besides these standard artifacts included in the package, it also allows Gateway implementers to store custom configurations to specify GraphQL WAF settings and more.
+
+![Simple Deployment Pipeline](pipeline-1.png)
+
+## Apollo Federation
+
+We recognize that some of you may have opted into Apollo Federation as a solution, and that is why we designed the Fusion schema composition so that it can compose any Apollo Federation subgraph into a Fusion subgraph. This allows you to seamlessly migrate from an Apollo Federation setup to a GraphQL-Fusion setup without the need to rewrite anything.
+
+## Conclusion
+
+We are still working on GraphQL-Fusion, but you can try it today with the Hot Chocolate Fusion Gateway. We plan to release the GraphQL-Fusion spec as it matures later this year under the **MIT license**. While the current iterations are primarily concerned with GraphQL, The Guild has begun specifying the Open API to GraphQL transformation they've developed in Mesh, and we'll integrate this as it becomes available. As mentioned before, we aim for the GraphQL-Fusion spec is to create an open specification that we will propose to the GraphQL Foundation Composite Schemas working group.
```

Comment:
The Guild has begun specifying the Open API to GraphQL transformation they've developed in Mesh, and we'll integrate this as it becomes available. WunderGraph wants to contribute a spec for Kafka (AsyncApi) and gRPC.